protocol/chat.go

```diff
@@ -288,6 +288,14 @@ func (c *Chat) IsActivePersonalChat() bool {
 	return c.Active && (c.OneToOne() || c.PrivateGroupChat() || c.Public()) && c.CommunityID == ""
 }
 
+func (c *Chat) DefaultResendType() common.ResendType {
```

Comment:
@cammellos, the function `protocol#GetResendTypeForChat` seems to be a duplicate of this new function you just defined. Shouldn't the places `GetResendTypeForChat` is called also use the new implementation you defined that caters for private group chats? If so, probably better would be to remove/replace `GetResendTypeForChat` with your `DefaultResendType`?

Reply:
oh yes, I didn't notice it was already there, I assumed it wasn't, I'll use mine since it's tested :)